doc/develop/modules.rst

```diff
@@ -761,6 +763,36 @@ are maintained in the module. For example:
     boards:
       - boards
 
+.. _modules-bin-blobs:
+
+Binary Blobs
+============
+
+Zephyr supports fetching and using :ref:`binary blobs <bin-blobs>`, and their
+metadata is contained entirely in :file:`zephyr/module.yml`. This is because
+a binary blob must always be associated with a Zephyr module, and thus the
+blob metadata belongs in the module's description itself.
+
+Binary blobs are fetched by :ref:`west <west-blobs>`, there is no support for
+fetching binary blobs when not using west.
```

Comment:
How about.
```suggestion
Binary blobs can be fetched using :ref:`west <west-blobs>`.
If ``west`` is not used, binary blobs must be downloaded and verified manually.
```

Reply:
Good point, forgot about this one. Will add.